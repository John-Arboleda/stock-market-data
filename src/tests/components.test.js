import renderer from 'react-test-renderer';
import MockAppComponent from './__mocks__/App';
import '@testing-library/jest-dom';

describe('Render App', () => {
  test('Assert snapshot', () => {
    const appSnap = renderer.create(
      <MockAppComponent />,
    ).toJSON();
    expect(appSnap).toMatchSnapshot();
  });
});
