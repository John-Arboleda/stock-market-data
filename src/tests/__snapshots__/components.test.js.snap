// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App Assert snapshot 1`] = `
Array [
  <nav
    className="navbar navbar-expand navbar-dark bg-primary"
  >
    <div
      className="d-flex justify-content-between container"
    >
      <div
        className="navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rr-ui-event-key="/"
          disabled={false}
          href="/"
          onClick={[Function]}
        >
          <svg
            fill="white"
            height={25}
            viewBox="0 0 16 16"
            width={25}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1 8a7 7 0 1 0 14 0A7 7 0 0 0 1 8zm15 0A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-4.5-.5a.5.5 0 0 1 0 1H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5z"
              fillRule="evenodd"
            />
          </svg>
        </a>
      </div>
      <a
        className="navbar-brand"
        href="/"
      >
        <h2
          className="m-0"
        >
          Stock Market Watcher
        </h2>
      </a>
      <div
        className="navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <svg
            fill="white"
            height={25}
            viewBox="0 0 16 16"
            width={25}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5 3a3 3 0 0 1 6 0v5a3 3 0 0 1-6 0V3z"
            />
            <path
              d="M3.5 6.5A.5.5 0 0 1 4 7v1a4 4 0 0 0 8 0V7a.5.5 0 0 1 1 0v1a5 5 0 0 1-4.5 4.975V15h3a.5.5 0 0 1 0 1h-7a.5.5 0 0 1 0-1h3v-2.025A5 5 0 0 1 3 8V7a.5.5 0 0 1 .5-.5z"
            />
          </svg>
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <svg
            fill="white"
            height={25}
            viewBox="0 0 16 16"
            width={25}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.405 1.05c-.413-1.4-2.397-1.4-2.81 0l-.1.34a1.464 1.464 0 0 1-2.105.872l-.31-.17c-1.283-.698-2.686.705-1.987 1.987l.169.311c.446.82.023 1.841-.872 2.105l-.34.1c-1.4.413-1.4 2.397 0 2.81l.34.1a1.464 1.464 0 0 1 .872 2.105l-.17.31c-.698 1.283.705 2.686 1.987 1.987l.311-.169a1.464 1.464 0 0 1 2.105.872l.1.34c.413 1.4 2.397 1.4 2.81 0l.1-.34a1.464 1.464 0 0 1 2.105-.872l.31.17c1.283.698 2.686-.705 1.987-1.987l-.169-.311a1.464 1.464 0 0 1 .872-2.105l.34-.1c1.4-.413 1.4-2.397 0-2.81l-.34-.1a1.464 1.464 0 0 1-.872-2.105l.17-.31c.698-1.283-.705-2.686-1.987-1.987l-.311.169a1.464 1.464 0 0 1-2.105-.872l-.1-.34zM8 10.93a2.929 2.929 0 1 1 0-5.86 2.929 2.929 0 0 1 0 5.858z"
            />
          </svg>
        </a>
      </div>
    </div>
  </nav>,
  <div
    className="p-0 container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <div
          className="card bg-primary text-white"
        >
          <div
            className="card-body"
          >
            <div
              className="p-3 text-center"
            >
              <p
                className="p-3 text-center card-text"
              >
                Stock Market Watcher allows you to search over 
                0
                <span>
                   
                </span>
                from different sectors with a combiened market cap of
                <span>
                   
                </span>
                $ 0.00 B
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="g-0 row row-cols-md-3 row-cols-2"
    />
  </div>,
]
`;
